docs/modules/plugins/partials/mobile-android-emulator.adoc

```diff
@@ -0,0 +1,19 @@
+====
+
+. Open Android Studio
```

Comment:
I think these docs cover it? https://developer.android.com/studio/run/managing-avds